### PR TITLE
MCS-38 - Autoformatting for scene generator code

### PR DESCRIPTION
**Pull request by rartiss55**
Figured this would be a big enough PR on its own. This includes the initial run of autopep8 on the scene generator code + the vscode settings file. Still going through and fixing linter errors manually that autopep8 didn't get (that will be another PR), so I didn't add the scene gen code to the pre commit hook yet. 

I would like to get this merged in by early next week so that there (hopefully) won't be too many merge conflicts going forward.

### CODE REVIEW

File: scene_generator/containers.py [hunk 1]
```diff
@@ -7,14 +7,23 @@
...
+def put_object_in_container(obj_def: Dict[str,
+                                          Any],
+                            obj: Dict[str,
+                                      Any],
+                            container: Dict[str,
+                                            Any],
+                            container_def: Dict[str,
+                                                Any],
+                            area_index: int,
+                            rotation: Optional[float] = None) -> None:
```
Comment (ThomasSchellenbergNextCentury) [inline scene_generator/containers.py:19]: This is weird enough to give me pause.  Was this done by the `black` auto formatter?

Comment (rartiss55) [reply scene_generator/containers.py:19]: Nope, this was autopep8.

Comment (rartiss55) [reply scene_generator/containers.py:19]: I can see why its not ideal but I started thinking of this ticket as a first pass -- we probably want to take the linter/formatter further, but we can't do that until we address all our warnings and errors.

Comment (ThomasSchellenbergNextCentury) [reply scene_generator/containers.py:19]: Great.  @deanwetherby can we change this?  This is really ugly.

Comment (rartiss55) [reply scene_generator/containers.py:19]: Okay, so it looks like if I decrease the aggressiveness level this won't happen (only caveat being I'll have to fix these types of line length issues by hand).

**Comment by rartiss55** [comment]
Closing...its easiest to just rerun the autofixer to address the issue above.

**Pull request closed without merging**